rps.js

```diff
@@ -0,0 +1,24 @@
+const playGame = function (userChoice) {
+    userChoice = userChoice.toUpperCase();
+
+    let computerChoice = Math.floor(Math.random() * 3) + 1;
+    if( computerChoice == 1) {
+        computerChoice = 'Rock';
+    } else if(computerChoice ==2) {
+        computerChoice = 'Paper';
+    } else {
+        computerChoice = 'SScissors';
```

Comment:
SScissors is causing bugs in your code 

be more consistent with your spelling